scripts/monorepo/getDependencies.js

```diff
@@ -16,8 +26,9 @@ function flattenPackageGraph(rootPackages, projectGraph, packageList = []) {
 /**
  * Returns all the dependencies of a given package name
  * @param {string} packageName including `@fluentui/` prefix
- * @param {string} options.dev include dev dependencies
- * @param {string} options.production include production dependencies
+ * @param {Object} options
+ * @param {string=} options.dev include dev dependencies
```

Comment:
```suggestion
 * @param {string} options.dev include dev dependencies
```

Reply:
this is incorrect as all properties are optional, speaking of which I forgot to mark `production` optional as well.

Reply:
fixed https://github.com/microsoft/fluentui/pull/24345/commits/d54541b988dd193144631652ccc2fd268819a1f7#diff-52ddc0ac2e681b9add19d5cb346d5bd5f4bf62b83952c7d1a66e77851e324849R29-R31